diff -ur coda-6.0.6-orig/coda-src/kerndep/pioctl.h coda-6.0.6/coda-src/kerndep/pioctl.h
--- coda-6.0.6-orig/coda-src/kerndep/pioctl.h	2003-05-23 12:26:31.000000000 -0600
+++ coda-6.0.6/coda-src/kerndep/pioctl.h	2004-06-29 01:57:25.082839231 -0600
@@ -100,7 +100,7 @@
 
 
 /* people who understand ioctling probably know why this is useful... */
-#define _VICEIOCTL(id)  (_IOW('V', id, struct ViceIoctl))
+#define _VICEIOCTL(id)  (_IOW_BAD('V', id, struct ViceIoctl))
 #define _VALIDVICEIOCTL(com) (com >= _VICEIOCTL(0) && com <= _VICEIOCTL(255))
 
 
